.github/workflows/stale.yml

```
@@ -29,10 +29,9 @@ jobs:
           stale-pr-label: "stale"       # label to use when marking as stale
 
           stale-pr-message: >
-            This PR had no visible activity in the past 60 days, labeling it as stale.
-            Any new activity will remove the stale label. To attract more reviewers, please tag
-            someone or notify the dev@solr.apache.org mailing list.
+            This PR has had no activity for 60 days and is now labeled as stale. 
+            Any new activity or setting it as a draft will remove the stale label. 
+            To attract more reviewers, tag someone or notify the dev@solr.apache.org mailing list. 
```

Maybe we could broaden from '(some)one' to 'several' (which might be less or more daunting depending on point of view) and say a little more about how to identify people to tag?
```suggestion
            To attract more reviewers, tag people who might be familiar with the code area and/or notify the dev@solr.apache.org mailing list. 
```